docker/default.nix

```diff
@@ -100,7 +101,7 @@ let
     services.cardano-explorer = {
       enable = true;
       cluster = environment;
-      socketPath = "/run/cardano-node/node-core-0.socket";
+      socketPath = "/run/cardano-node/node-core-CoreNodeId 0.socket";
```

Comment:
This change is probably accidental. But we should get rid of the use of node ids in paths completelty.